boards/arm/disco_l475_iot1/disco_l475_iot1.dts

```diff
@@ -118,13 +118,13 @@
 &spi3 {
 	status = "okay";
 
-	cs-gpios = <&gpiod 13 0>, <&gpioe 0 0>;
+	cs-gpios = <&gpiod 13 (GPIO_ACTIVE_HIGH | GPIO_PULL_UP)>, <&gpioe 0 0>;
```

Comment:
I overlooked it earlier, do we need to add flags also to `gpioe`? Also, this change doesn't seem to be related to hci-spi driver. Maybe we should do it in another PR?

Reply:
`gpioe` is dealt with in https://github.com/zephyrproject-rtos/zephyr/pull/21066.

I see it related in the fact that it deals with the same global functionality which is hci BLE  over SPI. So I have been able to test and validate porting to new API in current change. I don't see much value in splitting it.